content/en/security/application_security/api-inventory/_index.md

```diff
@@ -31,7 +31,12 @@ The following library versions are compatible with API Security Inventory. [Remo
 |Technology|Minimum version| Support for sensitive data scanning |
 |----------|----------|----------|
 |Python    | v2.1.6   | Requests and responses |
-|Golang    | v1.59.0  | Requests |
+|Java      | v1.31.0  | Requests only |
+|PHP      | v0.98.0  | Requests and responses |
+|.NET Core | v2.42.0  | Requests and responses |
+|.NET Fx   | v2.47.0  | Requests and responses |
+|Ruby      | v1.15.0  | Requests only |
+|Golang    | v1.59.0  | Requests only |
 
```

Comment:
We maintain three release lines, v5, v4 and v3 depending on the nodejs runtime so

```suggestion
|Node.js | v5.6.0, v4.30.0, v3.51.0 | Requests and responses |
```